device-identifier/shbdeviceidentifier/utilities/capture_utilities.py

```diff
@@ -21,49 +23,73 @@ def _convert_Capture_to_DataFrame_JIT(cap, num_of_packets, progress_proxy) -> pd
     The function is JIT compiled using numba.
     The function is not intended to be used directly, but within a tqdm progress bar.
     """
-    src_series = np.empty((num_of_packets,), dtype=object)
-    dst_series = np.empty((num_of_packets,), dtype=object)
-    ts_series = np.empty((num_of_packets,), dtype=pd.Timestamp)
-    protocol_series = np.empty((num_of_packets,), dtype=object)
-    data_len_series = np.empty((num_of_packets,), dtype="uint16")
+    src_addresses = np.empty((num_of_packets,), dtype=object)
+    src_ips = np.empty((num_of_packets,), dtype=object)
+    src_ports = np.empty((num_of_packets,), dtype=int)
```

Comment:
_No action required, but as a note:_ `dtype=int` works, but should be `object` since ports do not have a defined order relation. That being said, `int` is likely to be preferred from a programmatic / efficiency point of view. Are there other reasons for using `int` (or `object` respectively)?

Reply:
I only considered efficiency compared to `object`.